main.css

```diff
@@ -3,8 +3,10 @@ body {
 }
 .card1 {
     background-color: darkred;
-    border: dotted goldenrod 20px ;
-    color: aliceblue;
+    background-image: url(Ilvermorny_Crest_4.webp);
```

Comment:
`webp` images are not supported by all browsers yet. So you might have issues with this on browsers other than chrome.